src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  --default-color: #00acdc;
```

Comment:
```suggestion
  --primary-color: #00acdc;
```

Reply:
Also, use [root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) for declaring variables